{"name": "test_a", "status": "failed", "statusDetails": {"message": "AssertionError: assert False", "trace": "def test_a():\n>       assert False\nE       assert False\n\ntest_xdist.py:17: AssertionError"}, "start": 1612408627097, "stop": 1612408627097, "uuid": "f5486cc9-bb3a-4b9c-a39d-c9bd6851215a", "historyId": "bbfa62b961df2f3119ba3af9ba885754", "testCaseId": "db92e6a35aeb398cab57cd866b663bf6", "fullName": "test_xdist#test_a", "labels": [{"name": "suite", "value": "test_xdist"}, {"name": "host", "value": "wangwei"}, {"name": "thread", "value": "5544-MainThread"}, {"name": "framework", "value": "pytest"}, {"name": "language", "value": "cpython3"}, {"name": "package", "value": "test_xdist"}]}